packages/ui/client/composables/module-graph.ts

```diff
@@ -63,6 +63,8 @@ export function useModuleGraph(data: Ref<{
       .flatMap(([module, deps]) => deps.map((dep) => {
         const source = nodeMap[module]
         const target = nodeMap[dep]
+        if (source === undefined || target === undefined)
+          return undefined
```

Comment:
Unfortunately, the graph once again contains invalid data so filtering is necessary again.